tapestry.js

```diff
@@ -2026,6 +2164,37 @@ function exitViewMode() {
     startForce();
 }
 
+function generateTextNodeHTML(title, str) {
+    var lightboxContent = document.createElement("div");
+
+    var titleSection = document.createElement("div");
+    titleSection.setAttribute("id", "text-light-box-title");
+
+    var titleText = document.createElement("p");
+    titleText.appendChild(document.createTextNode(title));
+    titleSection.append(titleText);
+    lightboxContent.append(titleSection);
+
+    if (str) {
+        var paragraphSection = document.createElement("div");
+        paragraphSection.setAttribute("id", "text-light-box-paragraph");
+        paragraphArray = str.split("\n");
+        for (var i = 0; i < paragraphArray.length; i++) {
+            if (paragraphArray[i] !== "") {
```

Comment:
Let's remove this as users may be adding in a blank line on purpose.

Reply:
Will add line break on else statement